documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+[NOTICE](#NOTICE)
+[Template](#Template)
+[Comments](#Comments)
+
+## DEVNOTES
+
+feature/184 - version 0.2 documentation
+Working on formatting this document like the [spawnPatterns readme.](openscope/src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md).
+Plans:
+    - Remove all "// <explanation>" lines from the template so it can be copy/pasted.
```

Comment:
@indianbhaji my intent for this doc is to be an outline of what is expected, and what everything is supposed to do. something like (https://github.com/openscope/openscope/blob/feature/184/src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md).

I think the copy/pasted part is more of a note.  I don't intend for airport makers to actually copy paste the code from this document for building a new airport.  Though they should use it as a guide and they should be able to refer to this doc as a reliable source of truth when building a new airport file.